src/oatpp-postgresql/mapping/PgArray.hpp

```diff
@@ -0,0 +1,59 @@
+//
+// Created by dsmyth on 1/1/21.
```

Comment:
Hey @dsmyth ,

Would you mind putting oatpp file-header here. 
Also, please insert your name in that header.

Please send this change as a separate PR.

```
/***************************************************************************
 *
 * Project         _____    __   ____   _      _
 *                (  _  )  /__\ (_  _)_| |_  _| |_
 *                 )(_)(  /(__)\  )( (_   _)(_   _)
 *                (_____)(__)(__)(__)  |_|    |_|
 *
 *
 * Copyright 2018-present, Don Smyth <your-mail@mail.com>
 *                         Leonid Stryzhevskyi <lganzzzo@gmail.com>
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 ***************************************************************************/
```

Reply:
@dsmyth
Please also feel free to add your name to other files that you've touched.